src/webui/www/private/scripts/contextmenu.js

```diff
@@ -315,6 +315,8 @@ window.qBittorrent.ContextMenu ??= (() => {
             let all_are_super_seeding = true;
             let all_are_auto_tmm = true;
             let there_are_auto_tmm = false;
+            let thereAreV1Hashes = false;
```

Comment:
AFAIK a torrent can only have a single hash of each type, so these variables should be singular (e.g. `hasV1Hash` and `hasV2Hash`).

Reply:
thereAreV1/V2Hashes [among selected torrents], so everything is good I think, right?

Reply:
Ah right, multiple torrents can be selected. Disregard my original comment.